tests/utils/test_serializers.py

```diff
@@ -93,6 +95,31 @@ def serialize_foo(value: Foo) -> str:
     assert not serializers.has_serializer(Foo)
 
 
+class TestEnum(str, Enum):
+    """TestEnum."""
+
+    FOO = "foo"
+    BAR = "bar"
+
+
+class TestEnum2(Enum):
```

Comment:
```suggestion
class FooEnumWithPrefix(Enum):
```

2 suggestions here

* remove the `Test` prefix from the name, so pytest doesn't try to think this is a test class (also do this for `TestEnum` -> `FooEnum`)
* Add `WithPrefix` to the end to help visually distinguish why these otherwise identical classes are different (this would particularly help in the parametrizations, which I had to re-read like 3 times before I noticed the `2`)